scripts/lang/fr/__init__.py

```diff
@@ -697,6 +698,9 @@ def last_template_handler(
         >>> last_template_handler(["Légifrance", "base=CPP", "numéro=230-45"], "fr")
         ''
 
+        >>> last_template_handler(["ar-mot", "elHasan_"], "fr")
+        '<span style="line-height: 0px;"><big><big>الحَسَن</big></big></span> <small>(elHasan_)</small>'
```

Comment:
The `<big>` tag is deprecated and no more used in HTML5, cf https://developer.mozilla.org/en-US/docs/Web/HTML/Element/big. Should we use something like `<span style="font-size:larger">...</span>"`?

Reply:
big is gone. Again, no idea how this span will look on a device.